template/README.md

```diff
@@ -0,0 +1,136 @@
+# Project XYZ Template
+
+This is a template for Project XYZ that follows good Python practices. Follow the [good practices](/README.md).
+
+## Minimum Requirements
+
+1. Use the specified linters: isort, black, and pylint.
+2. Use the specified documentation check with pydocstyle.
+3. Ensure test coverage is above 80%.
+4. Distribute your library as a package.
+
+## Folder Structure
+
+```
+project_template/
+├── src/
```

Comment:
Here, you list the directory called "src". However, this directory does not exist in the repository. I would not use a src directory. Instead, adapt the README to match the actual directory structure.